geotrek/api/v2/serializers.py

```diff
@@ -502,13 +510,8 @@ def get_departure_city(self, obj):
             return city.code if city else None
 
     class TouristicEventSerializer(TouristicModelSerializer):
-        organizer = serializers.SlugRelatedField(
-            read_only=True,
-            slug_field='label'
-        )
-        organizer_id = serializers.PrimaryKeyRelatedField(
-            read_only=True
-        )
+        organizer = serializers.SerializerMethodField(source="organizers")
```

Comment:
Si organizer est une liste, nomme le au pluriel
```suggestion
        organizers = serializers.SerializerMethodField()
```
Par contre, quid de la retrocompat ?
l'aggregateur utilise l'api v2, i lfaut qu'on puisse importer les données avec

Reply:
Merci pour les retours.
J'avais garder le "organizer" au singulier justement pour la rétrocompatibilité. Mais vu que le champs en maintenant en M2M, de toute façon il va falloir revoir pleins de choses.
Je me proposais de revoir aussi la partie parser, pour assurer la retrocompat

Reply:
Oui, on utilise aussi la propriété "organizer" dans Geotrek-rando-v3, c'est pour ça qu'on ne l'avait pas renommé.
Mais de mémoire, on peut faire un synonyme pour gérer la transition et que "organizers" ou "organizer" fonctionne ?